nautobot/core/api/utils.py

```diff
@@ -233,6 +222,7 @@ def format_output(field, field_value):
     return data
 
 
+# TODO: This is part of the drf-react-template work towards auto-generating create/edit form UI from the REST API.
```

Comment:
I don't think that this was part of of the template stuff, but doesn't matter, we'll shore it up.

Reply:
It's only called from `FormFieldsViewSetMixin.form_fields()`, so I think it falls under that umbrella?